src/blocks/Listing/Listing.scss

```diff
@@ -39,6 +39,10 @@
             color: $link;
         }
 
+        p {
+            color: black;
+        }
```

Comment:
please use the variable color, 
```suggestion
        color: $black;
```

Reply:
Thank you @zahraard changed to $black variable color